lib/ui/view/dm_chat_view/dm_jump_to_viewmodel.dart

```diff
@@ -18,6 +19,7 @@ class DmJumpToViewModel extends FormViewModel {
   void setFormStatus() {}
 
   final log = getLogger('DmJumpToViewModel');
+
   TextEditingController _controller = TextEditingController();
```

Comment:
Remove both TextEditingControllers... If you don't know how to, check filled stacks.com

Reply:
done